test/apps/iodemo/io_demo.cc

```diff
@@ -565,9 +567,15 @@ class DemoClient : public P2pDemoCommon {
     }
 
     bool run() {
-        UcxConnection* conn = connect();
-        if (!conn) {
-            return false;
+        UcxConnection* conn[opts().num_servers];
+        for (int i = 0; i < opts().num_servers; i++) {
+            conn[i] = NULL;
+        }
+        for (int i = 0; i < opts().num_servers; i++) {
+            conn[i] = connect(opts().server_addrs[i]);
+            if (!conn[i]) {
+                return false;
```

Comment:
could you please add a print to the user to tell to which server IP the client failed to connect? (and why, if possible)

Reply:
@alinask Added failed server IP. It's not easy to add failed reason here, but we already have error log in connect function can report failed reason.